src/schemas/auth.js

```diff
@@ -79,10 +79,21 @@ const updateRolePermissions = Joi.object({
   }).required(),
 });
 
+const removeAccessTokens = Joi.object({
```

Comment:
@AbegaM - we don't need all these fields to be provided in a logout request, do we?

Reply:
We have a built in validator in `src/middlewares/validation.js` and it requires some request parameters like `query`, `params` and `body`,that is the reason we are passing this objects to the validator middleware (To reuse it)

Reply:
@IanMayo is right we should consider updating the schema validator to only need the required fields in the future

Reply:
Ooh, I didn't fully understand your comment @AbegaM - are you saying we need this validation because the logic goes somewhere else after `removeAccessTokens` - so we need to check the contents?   I see `removeAccessToken` as the end of a logic flow though, I don't expect there to be any other database events/interactions after `log-out`.  (well, not until a new login).